### DISCUSSION TITLE
Restore mbox from another account's backup?

### DISCUSSION THREAD

**Discussion by msimcich** [GAM-team/got-your-back#373] [Q&A]
I have created an mbox backup from a google workspace account which I would like to restore to a standard gmail account. Is this possible with GYB? The restore instructions include this:
...the specified folder should exist and should have been used in a previous GYB backup. 
the destination account has never had a GYB backup. Do I need to do that, even if I don't use that backup for anything?

**Comment by msimcich** [comment]
I attempted this, restoring from a google takeout mbox backup. The last part of the gyb cmd feeback seemed to indicate success:
	The authentication flow has completed.
	Using backup folder D:\\google\20220321\archive01
then it seemed to exit from the gyb command and put me back to a plain command prompt
but I don't see any emails arriving in the gmail account. It's been an hour now.

**Comment by nicktrejo** [comment]
Hi @msimcich 
I have downloaded an export using Vault from user.X@mydomain.com to restore it in user.Y@mydomain.com. I unziped the file (twice, there was a zip in the zip), then excecuted the following:

`gyb --email user.Y@mydomain.com --action restore-mbox --local-folder user.X@mydomain.com-1\user.X@mydomain.com--user.X@mydomain.com_0.mbox --strip-labels --label-restored user.X@mydomain.com --service-account`

Beware that I am in Windows, and that there are specific flags that may not fit your needs.
Hope it helps you ! 😃

**Reply by msimcich** [reply]
OK, that's super helpful, appreciate it! I'm on windows too so that part is aligned.
